src/videojs-contrib-hls.js

```diff
@@ -134,6 +133,46 @@ export default class HlsHandler extends Component {
       externHls: Hls
     });
 
+    this.masterPlaylistController_.on('loadedmetadata', () => {
+      let audioTracks = this.tech_.audioTracks();
```

Comment:
I will just change the name re-use over the scope won't work to well. I guess I misunderstood.